tests/baselines/reference/emptyArrayBindingPatternParameter04.js

```diff
@@ -0,0 +1,11 @@
+//// [emptyArrayBindingPatternParameter04.ts]
+
+
+function f([] = [1,2,3,4]) {
```

Comment:
looking at this, i think it should be an error. this can be any arbitrary expression with side effects, and we will be ignoring it. What is the ES6 spec says about this?